components/web/apps/org.wso2.carbon.apimgt.store/src/main/public/js/application.js

```diff
@@ -1,3 +1,39 @@
+$(window).load(function () {
+
+    var client = new SwaggerClient({
+        url: 'https://apis.wso2.com/api/am/store/v0.10/swagger.json',

```

Comment:
This code has been moved to public/js/applications/index.js. Fixed in https://github.com/wso2/carbon-apimgt/commit/82f2e62ad13d082ec487bb416a69a72df180d322